### PR TITLE
project: implement binding for statObject()

### PR DESCRIPTION
**Pull request by Erikvv**

### CODE REVIEW

**Review by egonelbre — DISMISSED** [review]

File: test/StatObjectTest.php [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+        self::assertLessThan(
+            60,
+            abs(time() - $systemMetaData->getCreated()->format('U'))
+        );
+    }
+
+    // appearantly this isn't implemented
+//    public function testStatPrefix(): void
+//    {
+//        $objectInfo = Util::project()->statObject('phpunit', 'StatPrefix');
```
Comment (ifraixedes) [inline test/StatObjectTest.php:43]: I'm wondering what does it return?
does `StatPrefix/` produce the same result?

**Review by ifraixedes — COMMENTED** [review]
It looks good to me but I have a couple of questions.

File: test/StatObjectTest.php [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+        self::assertEquals(55, $systemMetaData->getContentLength());
+        self::assertEquals(null, $systemMetaData->getExpires());
+        self::assertLessThan(
+            60,
+            abs(time() - $systemMetaData->getCreated()->format('U'))
+        );
+    }
+
+    // appearantly this isn't implemented
```
Comment (mniewrzal) [inline test/StatObjectTest.php:40]: should we keep this commented test? I don't think we will be supporting prefixes in StatObject

File: test/StatObjectTest.php (hunk 1 above)
Comment (Erikvv) [reply test/StatObjectTest.php]: Good feedback. 

It returns an error that the object does not exist. Both with `/` and without.

I've uncomment the test, duplicated it for both cases, and added an assertion for the exception.

**Review by mniewrzal — APPROVED** [review]

File: test/StatObjectTest.php (hunk 2 above)
Comment (Erikvv) [reply test/StatObjectTest.php]: I've uncommented it so that it confirms the existing behavior.

**Review by mniewrzal — APPROVED** [review]

**Review by ifraixedes — APPROVED** [review]

**Pull request merged**